docs/team/peiyuan.md

```diff
@@ -2,33 +2,134 @@
 layout: page
 title: Lim Peiyuan's Project Portfolio Page
 ---
+### Introduction
 
-### Project: Financial Advisor Smart Tracker (FAST)
+I am a software developer for Financial Advisor Smart Tracker (FAST). I was in-charge of the Code Quality of FAST.
+At the time of developed FAST, I was a Year 2 Computer Science Undergraduate.
```

Comment:
Wording sounds a bit awkward here. Are we supposed to use past tense here? (Since we are technically working on the project now)

Reply:
But in one-week time, we will be done with this. So, it save some trouble of updating this in the future